python/singa/net.py

```diff
@@ -53,13 +53,20 @@
     y = net.predict(x)
     print tensor.to_numpy(y)
 """
-
+from __future__ import print_function
+from __future__ import absolute_import
+
+from future import standard_library
+standard_library.install_aliases()
+from builtins import zip
+from builtins import str
+from builtins import object
 from .proto.model_pb2 import kTrain, kEval
-from __init__ import __version__
-import tensor
-import layer
-import snapshot
-import cPickle as pickle
+from .__init__ import __version__
+from . import tensor
+from . import layer
+from . import snapshot
+import pickle as pickle
```

Comment:
In Python 3, cPickle is renamed to pickle.

This code is generated by python-future which has a limitation : 

"We don't currently support these modules
 import pickle     # should (optionally) bring in cPickle on Python 2"

http://python-future.org/_modules/future/standard_library.html